### DISCUSSION TITLE
Mojolicious session cookies with CORS - soon rejected because no "Partitioned" attribute

### DISCUSSION THREAD

**Discussion by nchaigne** [mojolicious/mojo#2178] [Q&A] [answered]
Hello,

I have a REST API managed by Mojolicious, hosted on a domain.
And a web application on (potentially) another domain, querying the Mojolicious REST API.

This works, having configured CORS.

However, I see warnings in Firefox console:
Cookie "mojolicious" will soon be rejected because it is foreign and does not have the "Partitioned" attribute.

I'm not sure what to do to solve this. It works for now, but as I understand it won't in a near future.
Has anyone faced this issue ? do you know what to do ?

**Comment by kraih** [comment, marked as answer]
We don't support the Partitioned attribute yet. Feel free to file a feature request or PR. https://datatracker.ietf.org/doc/html/draft-cutler-httpbis-partitioned-cookies#section-2.1

**Reply by nchaigne** [reply]
Thanks for your prompt answer!
I have opened a feature request as you suggested: https://github.com/mojolicious/mojo/issues/2179
